NoSoft-SA/nspack
# PS EDI out: allow to create a file for multiple party roles

Summary of this change:
- add secondary_party_role_ids column to edi_out_rules and edi_out_transactions tables and CRUD
-  PS EDI out: allow to create a file for multiple party roles (combination of party_role_id and secondary_party_role_ids)
   
### Added
- . multiple_parties: false to each of the flows in EDI_OUT_RULES_TEMPLATE in AppConst. Set it to true for EDI_FLOW_PS.
-  secondary_party_role_ids column to edi_out_rules and edi_out_transactions tables and CRUD

### Changed
- . PS EDI out: allow to create a file for multiple party roles (combination of party_role_id and secondary_party_role_ids)

How to test this code via the UI:
* add secondary_party_role_ids  to edi_out_rules
EDI | Config | Out rules

* create PS files for two marketers and then another for a combination of the two.
EDI | Actions | Send PS

* Check that all rows from both files are included in the combined file.
* Check the BT record - that the total counts and quantities in the combined file is a sum of the totals of the two single files.
EDI | Viewer | Upload a file
